awscli/autoprompt/output.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import argparse
+import io
+import re
+
+import jmespath
+from botocore.utils import ArgumentGenerator
+
+from awscli.formatter import get_formatter
+from awscli.autocomplete.local.fetcher import CliDriverFetcher
+
+
+class OutputGetter:
+    def __init__(self, driver):
+        self._cli_driver_fetcher = CliDriverFetcher(driver)
+        self._output_formats = driver.arg_table['output'].choices
+        self._session = driver.session
+        self._cache = {}
+        self._current_expression = None
+
+    def get_output(self, parsed):
+        operation_model = self._cli_driver_fetcher.get_operation_model(
+            parsed.lineage, parsed.current_command)
+        if operation_model:
```

Comment:
Not sure where this is happening now but for some command, it shows up as `No output` but after I add a space after the command the output shows up. Once command was: `aws deploy get-deployment` If I do not include a space, it suggests `No output`, but if I add a space `aws deploy get-deployment ` then it shows the normal output.

Reply:
Following up here... It looks like if the command is a substring of another command (e.g. `get-deployment` is a substring of `get-deployment-config`) that is when it shows up of having `No output` when toggling to the command that is a substring in the operation list. 

It looks like this is an issue with the doc panel as well. Is this something easy to fix?